File: Ad-Hoc/3060/3060.c
#include <stdio.h>

int main(){
    int v,p, resto, divisao;
    scanf("%d", &v);
    scanf("%d", &p);
    divisao = v/p;
    resto = v%p;
    if (resto == 0){
        for (int i = 0; i < p; i++){
            printf("%d\n", divisao);
        }
    }
    else {
        for(int j = 0; j < resto; j++){
            printf("%d\n", divisao+1);
        }
        for (int k = resto+1; k<=p; k++){
            printf("%d\n", divisao);
        }
    }
    return 0;
}

File: Ad-Hoc/3060/3060.py
V = int(input())
P = int(input())
if V>=10 and V<=1000 and P>=2 and P<=18:
    divisao = V//P
    resto = V%P
    if resto == 0:
        for _ in range(P):
            print(f"{divisao:.0f}")
    else:
        for x in range(resto):
            print(f"{divisao+1:.0f}")
        for y in range(resto+1,P+1):
            print(f"{divisao:.0f}")